tf-modules/archive/bootstrap.tf

```diff
@@ -0,0 +1,51 @@
+resource "aws_lambda_function" "custom_bootstrap" {
```

Comment:
Just to be clear I'm reading this correctly: this will add the custom bootstrap, then TF will run it prior to anything that depends on the data source?

Reply:
Yes. Unfortunately, the `ApiEndpoints` lambda function relies on the output of invoking the custom bootstrap, so it gets run every time you deploy.

Reply:
In the long run I'd like to see us remove the custom bootstrap Lambda function and just have the resources that it's managing created using Terraform.